src/blocks/scratch3_operators.js

```diff
@@ -31,8 +40,54 @@ Scratch3OperatorsBlocks.prototype.add = function (args) {
     return args.NUM1 + args.NUM2;
 };
 
+Scratch3OperatorsBlocks.prototype.subtract = function (args) {
+    return args.NUM1 - args.NUM2;
+};
+
+Scratch3OperatorsBlocks.prototype.multiply = function (args) {
+    return args.NUM1 * args.NUM2;
+};
+
+Scratch3OperatorsBlocks.prototype.divide = function (args) {
+    return args.NUM1 / args.NUM2;
+};
+
+Scratch3OperatorsBlocks.prototype.lt = function (args) {
+    return args.OPERAND1 < args.OPERAND2;
+};
+
 Scratch3OperatorsBlocks.prototype.equals = function (args) {
     return args.OPERAND1 == args.OPERAND2;
 };
 
+Scratch3OperatorsBlocks.prototype.gt = function (args) {
+    return args.OPERAND1 > args.OPERAND2;
+};
+
+Scratch3OperatorsBlocks.prototype.and = function (args) {
+    if (!args.OPERAND1 || !args.OPERAND2) {
+        return false;
+    }
+    return true;
```

Comment:
Ok, here was the reason:
<img width="223" alt="screen shot 2016-06-24 at 10 48 48 am" src="https://cloud.githubusercontent.com/assets/120403/16341067/4873acf4-39f9-11e6-86ca-6b05724111b0.png">
I think this is made better by just casting to Boolean, though, so I'll update the code to do that. Still maintain that we should do a careful audit :)